notebooks/network_analysis_sec.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = transport_stats.plot.bar(\n",
```

Comment:
Nice graph, some more suggestions here:

- Add ylabel including the unit, e.g. "Demand in TWh" (note: check if unit "TWh" is correct, you probably have to convert it to TWh first)
- add grid (something like https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.grid.html)